Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -110,29 +125,28 @@ final class PledgeCTAContainerView: UIView {
         _ = self?.pledgeCTAButton ?|> buttonStyleType.style
       }
 
-    self.viewModel.outputs.rootStackViewAnimateIsHidden
+    self.viewModel.outputs.pledgeCTAButtonIsHidden
+      .observeForUI()
       .observeValues { [weak self] isHidden in
-        let duration = isHidden ? 0.0 : 0.18
-        let alpha: CGFloat = isHidden ? 0.0 : 1.0
-        UIView.animate(withDuration: duration, animations: {
-          _ = self?.rootStackView
-            ?|> \.alpha .~ alpha
-        })
+        self?.animateView(self?.pledgeCTAButton, isHidden: isHidden)
       }
 
-    self.activityIndicator.rac.animating = self.viewModel.outputs.activityIndicatorIsAnimating
+    self.activityIndicatorContainerView.rac.hidden = self.viewModel.outputs.activityIndicatorIsHidden
+    self.pledgeCTAButton.rac.hidden = self.viewModel.outputs.pledgeCTAButtonIsHidden
+    self.pledgeRetryButton.rac.hidden = self.viewModel.outputs.pledgeRetryButtonIsHidden
+    self.subtitleLabel.rac.text = self.viewModel.outputs.subtitleText
     self.titleAndSubtitleStackView.rac.hidden = self.viewModel.outputs.stackViewIsHidden
     self.titleLabel.rac.text = self.viewModel.outputs.titleText
-    self.subtitleLabel.rac.text = self.viewModel.outputs.subtitleText
 
     // NB: vm needs to return button style
+
```

Comment:
Please remove the comment above this line, think it was just there as a reminder before.